GSA/digitalgov.gov
# POAM: April 2024

## Summary

This PR handles security vulnerabilities for April 2024

- Closes out -> https://github.com/GSA/digitalgov.gov/pull/7364
- [Change log](https://github.com/postcss/autoprefixer/blob/main/CHANGELOG.md) for autoprefixer
- [Change log](https://github.com/postcss/postcss/blob/main/CHANGELOG.md) for postcss
- [Change log](https://github.com/uswds/uswds/releases) for USWDS
- [Change log](https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md) for dotenv
- [Change log](https://github.com/sass/dart-sass/blob/main/CHANGELOG.md) for sass
- [Change log](https://github.com/sass/embedded-host-node/blob/main/CHANGELOG.md) for sass-embedded



DependenciesDependency | Old | New
-- | -- | --
 autoprefixer                | 10.4.16  | 10.4.19
postcss                         | 8.4.33  | 8.4.38
@uswds/uswds             | 3.7.1  | 3.8.0
dotenv                           | 16.3.1  | 16.4.5
sass                              |1.69.7  | 1.74.1
sass-embedded          | 1.69.7  | 1.74.1



### Solution

### How To Test

- Verify no visual changes on [preview build](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-poam-apr24/)
- Pull down branch and run `npm install` and ensure no build errors 
- Run `Hugo` in your cli and check that builds successfully
- Start local with `npm run start` and visit local and visually inspect site to ensure no regressions


---

### Dev Checklist
